layer/migration.go

```diff
@@ -14,30 +14,33 @@ import (
 	"github.com/vbatts/tar-split/tar/storage"
 )
 
-func (ls *layerStore) MountByGraphID(name string, graphID string, parent ChainID) (l RWLayer, err error) {
+// MountByGraphID creates a RWLayer in the layer store using
+// the provided name with the given graphID. To get the RWLayer
+// after migration the layer may be retrieved by the given name.
+func (ls *layerStore) MountByGraphID(name string, graphID string, parent ChainID) (err error) {
```

Comment:
As this doesn't mount any more, maybe we should call it `CreateRWLayerByGraphID`